radicle/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2021"
 [features]
 default = []
 test = ["quickcheck"]
+sql = ["rusqlite"]
```

Comment:
Nitpicking: “sql” sounds a bit imprecise to me.  How about “persistent_routing_table”?

Reply:
The crate feature is just enabling the `From/ToSql` instances